### PR TITLE
1.0.3

### PR DESCRIPTION
**Pull request by AppleDevelops**
MCPE 1.0.3 beta is out, shoghicp gave us 3rd party server softwares some love. Do not merge this yet as there may be more betas.

### CODE REVIEW

**Comment by remotevase** [comment]
YEAH!

File: src/pocketmine/command/SimpleCommandMap.php [hunk 1]
```diff
@@ -123,6 +124,7 @@ private function setDefaultCommands(){
 		$this->register("pocketmine", new TeleportCommand("tp"));
 		$this->register("pocketmine", new TimeCommand("time"));
 		$this->register("pocketmine", new TimingsCommand("timings"));
+		$this->register("pocketmine", new TransferCommand("transferserver"));
```
Comment (remotevase) [inline src/pocketmine/command/SimpleCommandMap.php]: "TransferServer"? How about just "transfer"?

Comment (AppleDevelops) [inline src/pocketmine/command/SimpleCommandMap.php]: It's how it is in vanilla mcpe

Comment (remotevase) [inline src/pocketmine/command/SimpleCommandMap.php]: Okay then!

**Comment by Driesboy** [comment]
make something that a player can transfer a other player

**Comment by AppleDevelops** [comment]
@Driesboy I'm trying to make it follow vanilla MCPE

**Comment by Driesboy** [comment]
ok

**Comment by remotevase** [comment]
New code name!

**Comment by AnkitM252** [comment]
@Driesboy you can make a plugin with the transfer packet! Make it like the transferserver command with the player option!

**Comment by Driesboy** [comment]
like fasttransfer???

**Comment by AnkitM252** [comment]
Yep

**Comment by Driesboy** [comment]
I start working on it

Op 3-feb.-2017 19:05 schreef "Ankit" <notifications@github.com>:

**Comment by AnkitM252** [comment]
@Driesboy i made one [here](https://github.com/AnkitM252/TransferPE)!

**Comment by remotevase** [comment]
Nice!

**Comment by remotevase** [comment]
V1.0.3 is rolling out on all devices later today
https://twitter.com/beta_mcpe1/status/829725867303718912
https://minecraft.net/en-us/article/candy-pack-sweetens-pocket-win-10

**Comment by remotevase** [comment]
Oh, this too! https://twitter.com/MCPEAppStore/status/829728141077839873

**Pull request merged**